profile/README.md

```diff
@@ -87,22 +84,22 @@ For more guidance, you can check out a full tutorial on [Creating Wolfi Images w
 Wolfi is a Linux _undistro_  designed from the ground up to support newer computing paradigms such as containers. Although Wolfi has a few similar design principles as Alpine (such as using apk), it is a different distribution that is  focused on supply chain security. Unlike Alpine, Wolfi does not currently build its own Linux kernel, instead relying on the host environment (e.g. a container runtime) to provide one.
 
 ### Is Wolfi free to use?
-Yes, Wolfi is free and will always be.
+Yes, Wolfi is freely [available under the Apache License V2](https://github.com/wolfi-dev/os/blob/main/LICENSE) and will always be.
 
 ### Can I mix packages from Alpine repositories into a Wolfi-based image? 
-No, it’s not possible to mix Alpine apks with Wolfi apks. If your image requires dependencies that are currently only available for Alpine, you might consider opening a new issue in the [wolfi-os](https://github.com/chainguard-dev/wolfi-os/) repository to suggest the new package addition, or use [melange](https://github.com/chainguard-dev/melange) to build a custom apk for your image.
+No, it’s not possible to mix Alpine apks with Wolfi apks. If your image requires dependencies that are currently only available for Alpine. You might consider using [melange](https://github.com/chainguard-dev/melange) to build a custom apk for your image.
 
 ### Can I use Wolfi on the Desktop?
-No. Desktop distributions require additional software that is out of scope for Wolfi's roadmap.
+No. Desktop distributions require additional software that is out of scope for Wolfi.
 
 ### Who maintains Wolfi?
-Wolfi was created and is currently maintained by [Chainguard](https://chainguard.dev).
+Wolfi was created and is currently maintained by [Chainguard](https://chainguard.dev) as the base of Chainguard OS.
 
 ### What are the plans for long-term Wolfi governance?
-We intend for Wolfi to be a community-driven project, which means over time it will have multi-vendor governance and maintainers. For now we're focused on building the project and community, and will revisit this in several months when a community has formed.
+We intend for Wolfi to be the core foundation of Chainguard OS and available as open source project for use with minimal packages as used in our free Chainguard Containers.
```

Comment:
My understanding is that they are different OSes but I have not been able to review many technical details.